### PR TITLE
Add text about making notification bar dismissable

### LINKED ISSUE #3379: Document how to add code to dismiss notification bar
We should add some help text at the top of `static/sass/blocks/_notification-bar.scss` to explain how to make a notification bar dismissible.

---

**Good First Issue**: This issue is good for first time contributors. If you've already contributed to Warehouse, please work on [another issue without this label](https://github.com/pypa/warehouse/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22good+first+issue%22) instead. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/). If you are working on this issue and have questions, please feel free to ask them here, [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [pypa-dev mailing list](https://groups.google.com/forum/#!forum/pypa-dev).

### PR DESCRIPTION
**Pull request by pgadige**
Fixes #3379 
Add help text to explain how to make a notification bar dismissable.

### CODE REVIEW

**Comment by pgadige** [comment]
I've submitted a work-in-progress[WIP] pull request. I'd like @nlhkabu @aalmazan to review the text that is added, and also provide suggestions about the grammar; sentence structure; content and the placement of text. I'll appreciate your valuable feedback. Thank you!

File: warehouse/static/sass/blocks/_notification-bar.scss [hunk 1]
```diff
@@ -33,6 +33,30 @@
     - dismissable: Indicates notification be be dismissed. Defaults to hidden.
```
Comment (nlhkabu) [inline warehouse/static/sass/blocks/_notification-bar.scss]: could we fix this typo at the same time "be be" should be "can be"

File: warehouse/static/sass/blocks/_notification-bar.scss [hunk 2]
```diff
@@ -33,6 +33,30 @@
...
     - visible: Indicates a visible, non-dismissed notification
 */
+/*
+  A notification bar can be made dismissable by adding the following to the main
```
Comment (nlhkabu) [inline warehouse/static/sass/blocks/_notification-bar.scss]: I think the placement of this is fine, however, I think you can include it in the same comment block :)

File: warehouse/static/sass/blocks/_notification-bar.scss [hunk 3]
```diff
@@ -33,6 +33,30 @@
...
+  notification-bar div:
+    - .notification-bar--dismissable to it's classes
```
Comment (nlhkabu) [inline warehouse/static/sass/blocks/_notification-bar.scss]: "it's" should be "its" https://data.grammarbook.com/blog/pronouns/1-grammar-error/

File: warehouse/static/sass/blocks/_notification-bar.scss [hunk 4]
```diff
@@ -33,6 +33,30 @@
...
+    - data-controller="notification"
+    - data-target="notification.notification" as attributes
```
Comment (nlhkabu) [inline warehouse/static/sass/blocks/_notification-bar.scss]: "as an attribute" ??

File: warehouse/static/sass/blocks/_notification-bar.scss [hunk 5]
```diff
@@ -33,6 +33,30 @@
...
+
+  The data-controller and data-target are used for handling visibility,
+  including, adding .notification-bar--visible to notification-bar div if it has
```
Comment (nlhkabu) [inline warehouse/static/sass/blocks/_notification-bar.scss]: no need for `,` after "including"

**Comment by nlhkabu** [comment]
Thanks @pgadige! A few comments from me, but otherwise it looks good! If @aalmazan can confirm the accuracy of the text, that would be great :)

**Comment by pgadige** [comment]
Thank you @nlhkabu for your valuable feedback. I've made the required changes. Please let me know of any suggestions after you've reviewed.

**Comment by nlhkabu** [comment]
Looks good to me @pgadige !  One small thing I picked up when reading it again - `Javascript` should be `JavaScript`.

**Comment by nlhkabu** [comment]
Thank you @pgadige ! Thanks for your review @aalmazan !

:tada: :man_dancing: :sparkles:

**Comment by brainwane** [comment]
Thanks @aalmazan and @pgadige! If either/both of you would like to work on one of these, please comment there to let us know:

* #2819
* #2921
* testing our PEP 566 support in #3299

**Pull request merged**